dags/vds_pull_vdsvehicledata.py

```diff
@@ -55,6 +56,15 @@
          template_searchpath=os.path.join(repo_path,'volumes/vds/sql'),
          schedule_interval='0 4 * * *') as dag: #daily at 4am
 
+    t_upstream_done = ExternalTaskSensor(
```

Comment:
I think this DAG would usually fail to find the upstream DAG done and would wait for the second retry. If you want to avoid this behavior, you can add a delay between the two DAGs (and update the schedule of one of the two DAGs to compensate for this delay as well)